atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPool.java

```diff
@@ -102,6 +103,7 @@
     Map<InetSocketAddress, CassandraClientPoolingContainer> currentPools = Maps.newConcurrentMap();
     final CassandraKeyValueServiceConfig config;
     final ScheduledThreadPoolExecutor refreshDaemon;
+    private final Optional<MetricRegistry> metricRegistry;
```

Comment:
How would that work? Should we break the API and force the clients to create a `MetricRegistry`, or would we have some type of default?